### PR TITLE
documentation pass 7

### PR DESCRIPTION
**Pull request by alganthe**
- Updated missionmaker -> modules
- Updated missionmaker -> classnames

- Changed ```<kbd>Left windows</kbd>``` to ```<kbd>⊞ Win</kbd>```
- Added concertina wire documentation (done)
- Added Dagr documentation (placeholder)
- Added Fonts documentation (doner)
- Added HuntrIR documentation (done)
- Added M2-XA documentation (done)
- Added sandbag documentation (done)
- Added spotting scope documentation (done)
- Added tactical ladder documentation (done)
- Added Yardage 450 documentation (done)
- Added Tripod documentation (done)

### CODE REVIEW

File: documentation/feature/dagr.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+layout: wiki
+title: Dagr
+group: feature
+parent: wiki
+---
+
+## 1. Overview
+
+Adds the defense Advanced GPS Receiver.
```
Comment (bux578) [inline documentation/feature/dagr.md]: `Defense`

File: documentation/feature/huntIR.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+layout: wiki
+title: HuntIR
+group: feature
+parent: wiki
+---
+
+## 1. Overview
+
+### 1.1 The HuntIR
+The **H**igh altitude **U**nit **N**avigated **T**actical **I**maging **R**ound (HuntIR) is designed to be fired from a grenade launcher, after being fired in the air the in built parachute will be deployed and the IR CMOS camera will activate, providing a video stream until it touches the ground or get shot down.
```
Comment (bux578) [inline documentation/feature/huntIR.md]: `... grenade launcher. After being ...`

File: documentation/feature/huntIR.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+## 2. Usage
+NOTE: the HuntIR round doesn't work with modded weapons without a compatibility fix made either by the ACE3 team or the mod team.
+
+### 2.1 Using the HuntIR
+- To be able to connect to the IR CMOS camera you'll need a `HuntIR monitor`.
+- Fire the HuntIR round as high as possible over the area you want to observe.
+- Open the `HuntIR monitor`.
+  - To open the `HuntIR monitor` self interact <kbd>CTRL</kbd> + <kbd>⊞ Win</kbd> (ACE3 default)
+  - Select `equipment`.
```
Comment (bux578) [inline documentation/feature/huntIR.md]: `Equipment`

File: documentation/feature/spotting_scope.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+## 1. Overview
+
+Adds a deployable spotting scope.
+
+## 2. Usage
+
+### 2.1 Deploying the spotting scope
+- Self interact <kbd>CTRL</kbd>+<kbd>⊞ Win</kbd> (ACE3 default).
+- Select `Equipment`.
+- Select `place spotting scope` (note that the scope will be at your feet).
```
Comment (bux578) [inline documentation/feature/spotting_scope.md]: `Place spotting scope`

File: documentation/feature/tripod.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+## 1. Overview
+
+Adds a packable tripod deployable on the field. It features a flat part to deploy your weapon on and adjustable legs.
+
+## 2. Usage
+
+### 2.1 deploying the tripod
+- Note that you need a `SSW kit` in your inventory.
```
Comment (bux578) [inline documentation/feature/tripod.md]: `SSWT kit`

**Comment by bux578** [comment]
Thank you. That's awesome.

**Comment by alganthe** [comment]
typos fixed.

**Comment by alganthe** [comment]
Fonts and tactical ladder doc added, ready for merging

**Comment by alganthe** [comment]
fixed and ready.

**Pull request merged**